docs/installation.rst

```diff
@@ -498,6 +498,8 @@ These platforms have been reported to work at the versions mentioned.
 | Operating system                 | | Tested Python           | | Latest tested  | | Tested     |
 |                                  | | versions                | | Pillow version | | processors |
 +==================================+===========================+==================+==============+
+| macOS 14 Sonoma                  | 3.8, 3.9, 3.10, 3.11      | 10.0.1           |arm           |
++----------------------------------+---------------------------+------------------+--------------+
 | macOS 13 Ventura                 | 3.8, 3.9, 3.10, 3.11      | 10.0.1           |arm           |
 |                                  +---------------------------+------------------+              |
 |                                  | 3.7                       | 9.5.0            |              |
```

Comment:
(the line before)
macOS 12 is actually Monterey )

Reply:
Ok, I've created #7431